notary-api/src/routes/index.js

```diff
@@ -3,5 +3,6 @@ import buyers from './buyers';
 import data from './data';
 import sellers from './sellers';
 import dataResponses from './dataResponses';
+import notaryInfo from './notaryInfo';
```

Comment:
`./notaryInfo` is missing.
moved to `src/routes/account.js` ?

Reply:
It's an `operation` now

Reply:
Wrongly deleted. Placed back.